### ISSUE TITLE
e3nn-gelib converter

### ISSUE THREAD

**Issue by mitkotak** [risi-kondor/GElib#11]
Trying to figure out a converter between e3nn and GElib. Just putting this here while I try to figure out the normalization difference

```python
import torch
import gelib
torch.manual_seed(0)
from e3nn import o3

def randn(irreps, batch=1, device='cpu'):
    return gelib.SO3vec.randn(batch, [mul for mul, ir in irreps], device=device)

irreps = o3.Irreps("0e + 1o")
x = randn(irreps)
z = gelib.CGproduct(x, x)
z = torch.cat([part.real.ravel() for part in z.parts])

x_e3nn = torch.cat([part.real.ravel() for part in x.parts])
z_e3nn = o3.FullTensorProduct(irreps, irreps)(x_e3nn, x_e3nn)

torch.testing.assert_close(z, z_e3nn)
```

**Comment by asglover** [comment]
I think it might be more than a normalization difference. I think the two libraries use a different sign convention. It's not the first time someone's used a different sign convention than e3nn. Let me know if it seems like I'm off base? I'll think about this, but maybe there's a straightforward way to resolve it? I wonder if's only off by a rotation fix rotation always? I'll play around with it tomorrow. 

![image](https://github.com/user-attachments/assets/f98409d2-94f5-4879-90fa-f74dc728b3bf)
